Sources/VGSCollectSDK/UIElements/Text Field/VGSTextField.swift

```diff
@@ -149,6 +149,8 @@ internal extension VGSTextField {
         // set main style for view
         mainStyle()
         // text view
+        textField.adjustsFontSizeToFitWidth = true
+        textField.minimumFontSize = 9.0
```

Comment:
We will make this attributes public in scope of other PR. Don't set default value for now.